conf.d/spark.yaml.example

```diff
@@ -0,0 +1,20 @@
+init_config:
+
+instances:
+  #
+  # The Spark check retrieves metrics from YARN's ResourceManager. This
+  # check must be run from the Master Node and the ResourceManager URI must
+  # be specified below. The ResourceManager URI is composed of the 
+  # ResourceManager's hostname and port.
+  #
+  # The ResourceManager hostname can be found in the yarn-site.xml conf file
+  # under the property yarn.resourcemanager.address
+  #
+  # The ResourceManager port can be found in the yarn-site.xml conf file under
+  # the property yarn.resourcemanager.webapp.address
+  #
+  - resourcemanager_uri: http://localhost:8088
+    cluster_id: MySparkCluster
```

Comment:
I think it is best to tag metrics with the `cluster_id`, provided by YARN, so all applications running on YARN can be grouped. I also think that the users should have the ability to give their cluster a friendly name, by adding the `cluster_name` in the YAML file.

I am not sure if it should be mandatory though. When making a tag mandatory in the YAML I see two options: providing a default value, and not providing a default value. If you provide a default value, that is great, it will work out of the box, but I am sure most users will not change the default value so everyone will have the tag set as the default value, e.g. `MySparkCluster`. If you don't provide a default value, it won't work out of the box, and users will have to debug the error message and/or read the instructions.

What is the preferred setting: mandatory with no default value?